src/setnpmreg.ps1

```diff
@@ -0,0 +1,8 @@
+If ((Test-Connection git.autodesk.com).PingSucceeded){
+    echo adsk reachable
+    New-Item -Path . -Name ".npmrc" -ItemType "file" -Value "registry=https://npm.autodesk.com/artifactory/api/npm/autodesk-npm-virtual/" -Force
+}
+else{
+    echo adsk not reachable
+    New-Item -Path . -Name ".npmrc" -ItemType "file" -Value "registry=https://registry.npmjs.org" -Force
+}
```

Comment:
Looks good. Lets get it in there and see if any problems pops up.

Reply:
nice